src/components/Sidebar/Header.jsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { propTypeTimebar } from '../../propTypes'
 
-const Header = ({ timebar: { rows } }) =>
-  <div className="sidebar__header">
-    {
-      rows.map(({ id, title }) =>
-        <div key={id} className="timebar-key">{title}</div>
-      )
-    }
+const Header = ({ isSticky, height, width, timebar: { rows } }) =>
+  <div style={isSticky ? { paddingTop: height } : {}}>
+    <div className={`sidebar__header ${isSticky ? 'is-sticky' : ''}`} style={isSticky ? { width } : {}}>
+      {
+        rows.map(({ id, title }) =>
+          <div key={id} className="timebar-key">{title}</div>
+        )
+      }
+    </div>
   </div>
 
 Header.propTypes = {
-  timebar: PropTypes.shape({})
+  isSticky: PropTypes.bool,
+  height: PropTypes.number,
+  width: PropTypes.number,
```

Comment:
I should account for `isSticky` being `true` when the component first renders, in PromComm it will be false so wouldn't have seen a bug.  I guess this may not be the case if a page was to load with the scroll position pre-set.